scripts/_common.sh

```diff
@@ -4,17 +4,29 @@
 # COMMON VARIABLES
 #=================================================
 
-YNH_PHP_VERSION="7.4"
-
-php_dependencies="php$YNH_PHP_VERSION-ctype php$YNH_PHP_VERSION-curl php$YNH_PHP_VERSION-mbstring php$YNH_PHP_VERSION-mcrypt php$YNH_PHP_VERSION-tokenizer php$YNH_PHP_VERSION-xml php$YNH_PHP_VERSION-xmlreader php$YNH_PHP_VERSION-xmlwriter php$YNH_PHP_VERSION-mysql"
-
-# dependencies used by the app (must be on a single line)
-pkg_dependencies="$php_dependencies mariadb-server"
-
 #=================================================
 # PERSONAL HELPERS
 #=================================================
 
+_ynh_agendav_find_caldav_app() {
+    mapfile -t all_apps < <(yunohost app list --output-as json --quiet | jq -r ".apps[].id")
```

Comment:
`mapfile` o.o

Reply:
Yeah and in the end I removed it because ` < <()` creates a sub-shell AND hides the return code, and thus breaks ynh_die… so now it's a simple string with cut -d " " .